src/elements/Image/Image.js

```diff
@@ -24,9 +24,7 @@ export default class Image extends Component {
     );
 
     return (
-      <div className={classes}>
-        <img {...this.props} />
-      </div>
+      <img {...this.props} className={classes} />
```

Comment:
since you are returning this one element, the parentheses are not necessary and you can move the image up to `line 26`.